src/ml/neural_net/style_transfer/mps_style_transfer_transformer_network.mm

```diff
@@ -165,4 +168,119 @@ - (MPSNNImageNode *) backwardPass:(MPSNNImageNode *) inputNode {
   return encoding1Grad;
 }
 
+- (void)setLearningRate:(float)lr {
+  [_encoding1 setLearningRate:lr];
+  [_encoding2 setLearningRate:lr];
+  [_encoding3 setLearningRate:lr];
+  [_residual1 setLearningRate:lr];
+  [_residual2 setLearningRate:lr];
+  [_residual3 setLearningRate:lr];
+  [_residual4 setLearningRate:lr];
+  [_residual5 setLearningRate:lr];
+  [_decoding1 setLearningRate:lr];
+  [_decoding2 setLearningRate:lr];
+  [[_conv weights] setLearningRate:lr];
+  [[_instNorm weights] setLearningRate:lr];
+}
+
+- (NSDictionary<NSString *, NSData *> *)exportWeights {
+  NSMutableDictionary<NSString *, NSData *> *weights;
+
+  NSDictionary<NSString *, NSData *> * encode1Weights = [_encoding1 exportWeights];
+
+  weights[@"transformer_conv0_weight"] = encode1Weights[@"conv_weights"];
```

Comment:
Abhishek, meet `-[NSMutableDictionary addEntriesFromDictionary:`. `-[NSMutableDictionary addEntriesFromDictionary:`, meet Abhishek. 😄 

https://developer.apple.com/documentation/foundation/nsmutabledictionary/1411035-addentriesfromdictionary?language=objc

Reply:
Oh wait, the keys are changing. Is there some way to do this more systematically?

For example, maybe some of the `exportWeights` methods can take a "prefix" argument?

Reply:
Perhaps, I wanted the output keys to match the output keys in the MxNet model. That's why the keys were changing.

Reply:
Converted to using the MPS model as the source of truth.